01_02_Combat_Rules.txt

```diff
@@ -999,3 +999,128 @@ retrieve a dropped weapon or stand back up if they have fallen.
+the characters in the lineup is important. Players can chose any position in the
+stackup order that hasn't yet been taken at the time that they declare their 
+character will stack up. So the first character to stack up could choose to get
+in the middle of the line, and the second character to use their turn to stack
+up could stack up in front if they so chose.  
+
+===== Breaching =====
+
+    Any time a character joins the stack up line, the character in the lead of
+the stack may call "breach!" At that time, regular initiative order is 
+interrupted, and the breach and clear phases occur during the turn of the 
+character who just got in line. After the clearing has resolved, play resumes
+after that character's turn.
+
+    Some weapons, tools or gadgets are commonly used during a breach maneuver.
+Examples include breaching charges, tear gas or flashbangs. If these are used,
+they take effect first in the breach. Keep in mind that a character using such
+a device will need a free hand to do so, and may need to spend one of their 
+clearing actions to re-draw their weapon. As such, devices are usually used by
+the characters in the 3rd or 4th positions.
+
+    Some rooms are defended with booby traps. In the case that a room or entry
+point has such perils, they take effect second. 
+
+    While devices are being resolved, all attackers and defenders roll a reflex
+save to determine the order of combat during the clearing phase. If the 
+defenders were unaware of the impending breach, then they roll reflex with a
+disadvantage and the attackers roll reflex with advantage.
+
+*Breaching Roll = Reflex Save + Weapon Reflex Mod + any mod from feats
+
+===== Clearing =====
+
+    Players take their turn in decending order of breaching roll. In other words,
+The character with the highest net reflex roll goes first, the character with 
+the lowest reflex goes last. Characters on the attacking team (the team that 
+stacked up) must use an Action to move from their position in the stack line
+before they can fire or perform melee attacks. Characters on the attacking team
+can take cover at the entry point that they were stacked up on, but doing so 
+will block the entry point and prevent another character from moving or firing
+through the entry point unless they are firing at that character.
+
+    The Clearing phase ends when either all of one team in the room can no 
+longer fight or the breaching order has been exhausted. Play resumes as normal
+after the turn of the last character to stack up after that.
+
+===== Interrupting a Stackup =====
+
+    If an enemy approaches or attacks one of the characters in a stack up line
+after they have stacked up but before the lead character has called "breach!",
+then all of the characters in the stack up are no longer considered to be 
+stacked up. Instead, they may use their held actions as immediate reactions in
+initative order and must wait until their next turn to stack back up.
```

Comment:
All of those held actions? I wonder whether there should there be some penalty for being interrupted while stacking, perhaps you only get one Reaction, or one reaction plus a cover dive like in the start of combat rules. This point is debateable.

Reply:
I was wondering that too as I wrote it, but was racing to get it all down so we had a complete system to iterate on. I like the one reaction + cover dive. I'll do that.